src/CLI/clitree/cli-xml/acl.xml

```diff
@@ -7,18 +7,16 @@
                         http://www.dellemc.com/sonic/XMLSchema/clish.xsd"
     >
 <!--=========================================================================-->
-
 <VIEW name="enable-view">
 <!--show ip access-lists -->
-   <COMMAND
-         name="show ip access-lists"
-         help="Show IPv4 access-list information"
-		 >
-         <PARAM
-            name="access-list-name"
-            help="Name of access-list (Max size 140)"
-            ptype="STRING"
-            optional="true"
+    <COMMAND name = "show ip access-lists"
+        help = "Show IPv4 access-lists information"
+        >
+        <PARAM 
+            name = "access-list-name"
```

Comment:
Nit: Can we try to keep spacing and line breaks consistent throughout the XML files? Makes it a bit easier to read.